sdk/ai/azure-ai-inference/azure/ai/inference/models/_enums.py

```diff
@@ -10,6 +10,15 @@
 from azure.core import CaseInsensitiveEnumMeta
 
 
+class AudioContentFormat(str, Enum, metaclass=CaseInsensitiveEnumMeta):
+    """A representation of the possible audio formats for audio."""
+
+    WAV = "wav"
```

Comment:
Often this is PCM, wav is container which can contain different formats like PCM, even MP3

Reply:
Discussed offline